internal/types/scheduled_task.go

```
@@ -187,6 +255,22 @@ type S3JobConfig struct {
+	RoleARN    string            `json:"role_arn,omitempty"`
+	ExternalID string            `json:"external_id,omitempty"`
+}
+
+func (s *S3JobConfig) ResolvedAccessMode() StorageAccessMode {
+	if s == nil || s.AccessMode == "" {
+		return StorageAccessModeStaticKey
+	}
+	return s.AccessMode
+}
+
+func (s *S3JobConfig) isGCS() bool {
```

golangci-lint (2.12.2)

[error] func (*S3JobConfig).isGCS is unused

(unused)